docs/gitbook/usage/metrics.md

```diff
@@ -401,3 +401,69 @@ Reference the template in the canary analysis:
           max: 5
         interval: 1m
 ```
+
+## Graphite
+
+You can create custom metric checks using the Graphite provider.
+
+Graphite template example:
+
+```yaml
+apiVersion: flagger.app/v1beta1
+kind: MetricTemplate
+metadata:
+  name: graphite-error-rate
+  namespace: ingress-nginx
+spec:
+  provider:
+    type: graphite
+    address: http://graphite.monitoring
+  query: |
```

Comment:
Any opinions on a good example query to provide?

Reply:
It may be worth wiring up a simple spring-boot app that has statsd configured so that we can provide a realistic query based on a "common" metric naming schema. I'll see if I can't do that quickly.